.eslintrc.js

```diff
@@ -9,8 +9,9 @@ module.exports = {
     "eslint:recommended",
     "plugin:ava/recommended",
     "plugin:@typescript-eslint/recommended",
-    "plugin:prettier/recommended",
-    "prettier/@typescript-eslint",
   ],
   ignorePatterns: ["dist/**/*"],
+  rules: {
+    "@typescript-eslint/no-explicit-any": "warn",
+  },
```

Comment:
`any` is used already in the project all over, but we werent running lint

have kept it as warnings